docs/rfc/rfc-1.adoc

```diff
@@ -41,9 +44,81 @@ and maintain the peg), we must turn to another token. Instead, the system pays
 operators in T tokens from a treasury funded by the T-token DAO and whenever it
 collects tBTC fees it uses those fees to buy T tokens back for its treasury.
 
+=== Minting And Unminting
+
+A user with an account balance can drain their account balance to mint TBTC.
+Similarly, a user can burn TBTC to refill their account balance.
+
+// FIXME: fill in this section with how much TBTC they can mint with regards to
+// their unswept balance
+
 == In Depth
 
-=== Minting
+[[group-size]]
+=== Group Size
+There are three main governable parameters that we need to choose:
+
+* The number of signers that can be controlled by the adversary before the key is in danger `threshold`
+* The number of group members required for a <<heartbeat,heartbeat>> to be considered "strong". `heartbeat`
+* The minimum number of members we're allowed to drop down to during the DKG
+  group formation re-try period. `dkg`
+* The total size of the group `groupSize`
+
+`threshold < heartbeat < dkg < groupSize`
+
+The total gas cost for sortiton increases linearly with group size by at least
+20k gas per member. Requiring more signers to sign messages makes it harder for
```

Comment:
With optimistic sortition we don't need to spend 20k gas per member in the happy path. If we use 32-bit operator IDs and pack them densely, the marginal cost per member would be an average of 2.5k gas from storing the ID. Additionally, we need at least `threshold + 1` signatures for the DKG result submission, which should be 3k gas per member, for a total of around 6k+change.

Reply:
Awesome! Is that documented somewhere so I can edit this into the RFC and point to it?

Reply:
Not yet, all the discussion here has been kind of eating up my time so I haven't had the focus to write up the sortition pool v2 spec

Reply:
Added a note in 0292903